server/agoraApi.yaml

```diff
@@ -309,6 +864,132 @@ components:
       type: array
       items:
         $ref: '#/components/schemas/Goal'
+    Topic:
+      type: object
+      required:
+        - topicName
+      properties:
+        id:
+          type: integer
+          description: Assigned automatically for new topics, used to update existing topic if provided
+          uniqueItems: true
+        topicType:
+          type: integer
+          description: The type
+        topicName:
+          type: string
+          description: The tag
+        topicDescription:
+          type: string
+          description: Description of the topic
+        topicImage:
+          type: string
+          description: filename of the associated image for the topic
+        topicHtml:
+          type: string
+          description: Embedded HTML
+        assessmentId:
+          type: integer
+          description: assessment tag
+        activityId:
+          type: integer
+          description: activity tag
+        hasAssessment:
+          type: boolean
+          description: Whether topic has assessment
+        hasActivity:
+          type: boolean
+          description: Whether topic has activity
+        active:
+          type: boolean
+          description: Active status, inactive items will not be returned in most queries
+        visibility:
+          type: integer
+          description: Public = 0, Shared = 1, Private = 0
```

Comment:
I believe public is visibility = 2, but double-check me on that.

Reply:
The default visibility when querying for getAllVisibleResources is 2, so this change makes sense. Good catch.